contracts/ColonyNetwork.sol

```diff
@@ -24,6 +24,7 @@ import "./IColony.sol";
 import "./EtherRouter.sol";
 import "./Token.sol";
 import "./ColonyNetworkStorage.sol";
+import "./ReputationMiningCycle.sol"; //TODO: Replace with interface
```

Comment:
The interface would be nice to carry over all public function natspec comments too. That will nicely shrink the main contract length and improve readability. Is it too complex to do it here?